<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_home">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>이전으로</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_Message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblNickname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>닉네임(이메일)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Nickname">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>391</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>보내기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
